torch/csrc/inductor/aoti_runtime/model.h

```diff
@@ -1,732 +1,9 @@
-// C ABI defined in torch/csrc/inductor/aoti_torch/c/shim.h. The same rule
-// applies to other files under torch/csrc/inductor/aoti_runtime/.
-#include <torch/csrc/inductor/aoti_runtime/device_utils.h>
-#ifdef USE_MPS
-#include <torch/csrc/inductor/aoti_torch/c/shim_mps.h>
-#endif // USE_MPS
-#ifdef USE_XPU
-#include <torch/csrc/inductor/aoti_runtime/utils_xpu.h>
-#else
-#include <torch/csrc/inductor/aoti_runtime/utils.h>
-#endif // USE_XPU
-#include <torch/csrc/inductor/aoti_runtime/constant_type.h>
-
-#define AOTI_RUNTIME_CHECK(EXPR, MSG) \
-  do {                                \
-    bool ok = EXPR;                   \
-    if (!ok) {                        \
-      throw std::runtime_error(MSG);  \
-    }                                 \
-  } while (0)
-
-// At codegen time, we write out a binary file called constants.bin.
-// We then turn the raw binary to an object file that exposes this
-// symbol and link it into the final .so.
-// For information on the binary format, see `man objcopy`, under
-// the "binary-architecture" flag:
-// https://man7.org/linux/man-pages/man1/objcopy.1.html
-// todo: use #embed in C++ 23 once available
-// The constants are NOT readonly because they may be mutated.
-// NOLINTNEXTLINE(*array*)
-extern uint8_t _binary_constants_bin_start[];
-// NOLINTNEXTLINE(*array*)
-extern uint8_t _binary_constants_bin_end[];
-
-#if defined(USE_CUDA) || defined(USE_XPU)
-// Compute required blob size with 64-alignment if on GPU.
-#define AOTI_CONST_ALIGNMENT 64
-#else
-// Use 64-alignment (use something >=64)for better performance on CPU.
-#define AOTI_CONST_ALIGNMENT 64
-#endif
-
-namespace {
-
-using RAIIDataPtr = std::unique_ptr<void, std::function<void(void*)>>;
-
-#ifdef USE_CUDA
-
-RAIIDataPtr RAII_gpuMalloc(size_t num_bytes) {
-  void* data_ptr;
-  AOTI_RUNTIME_DEVICE_CHECK(cudaMalloc((void**)&data_ptr, num_bytes));
-  auto deleter = [](void* ptr) { AOTI_RUNTIME_DEVICE_CHECK(cudaFree(ptr)); };
-  return RAIIDataPtr(data_ptr, deleter);
-}
-
-#elif defined(USE_XPU)
-
-RAIIDataPtr RAII_gpuMalloc(size_t num_bytes) {
-  sycl::queue* queue_ptr = nullptr;
-  aoti_torch_get_current_sycl_queue((void**)&queue_ptr);
-  void* data_ptr = sycl::malloc_device(num_bytes, *queue_ptr);
-  auto deleter = [queue_ptr](void* ptr) { sycl::free(ptr, *queue_ptr); };
-  return RAIIDataPtr(data_ptr, deleter);
-}
-
-#elif defined(USE_MPS)
-
-RAIIDataPtr RAII_gpuMalloc(size_t num_bytes) {
-  void* data_ptr = nullptr;
-  aoti_torch_mps_malloc(&data_ptr, num_bytes);
-  auto deleter = [](void* ptr) { aoti_torch_mps_free(ptr); };
-  return RAIIDataPtr(data_ptr, deleter);
-}
-
-#else
-
-RAIIDataPtr RAII_cpuMalloc(size_t num_bytes) {
-  void* data_ptr = std::malloc(num_bytes);
-  if (!data_ptr) {
-    throw std::bad_alloc();
-  }
-  auto deleter = [](void* ptr) { std::free(ptr); };
-  return RAIIDataPtr(data_ptr, deleter);
-}
-
-#endif // USE_CUDA
-
-} // anonymous namespace
+#include <torch/csrc/inductor/aoti_runtime/model_base.h>
 
```

Comment:
Please also copy the same WARNING here. All the header files in aoti_runtime have this warning just to play safe.